bfinst

```diff
@@ -171,15 +171,21 @@ install_grub()
     efidir=/mnt/boot
     bootdir=/mnt/boot
     localedir=/mnt/usr/share/locale
-    grubcfg=/mnt/boot/grub/grub.cfg
+    grubdir=/mnt/boot/grub
+    grubcfg=$grubdir/grub.cfg
 
     mount $ROOT_PART /mnt
     mount $EFI_PART /mnt/boot
 
     test "$(ls -A $efivars)" || mount -t efivarfs none $efivars
 
-    mkdir -p $localedir
-    grub-install $EFI_PART --locale-directory=$localedir --efi-directory=$efidir --boot-directory=$bootdir
+    mkdir -p $grubdir
```

Comment:
Another small question, what's the purpose of this line?

Reply:
Apparently the "grub-install" command creates the "$grubdir" directory. Since we don't run the "grub-install" we need to manually create that directory otherwise the below command "touch $grubcfg" will fail.

Reply:
Btw, $grubcfg is where grubaa64.efi is supposed to look for the configuration file.